web/crux-ui/locales/en/common.json

```diff
@@ -158,7 +158,9 @@
     "obsolete": "Obsolete",
     "unknown": "Unknown",
     "preparing": "Preparing",
-    "in-progress": "In progress"
+    "in-progress": "In progress",
+    "successful": "Successful",
```

Comment:
You should use `common:successful` and `common:failed` at the `t()` call instead.

Reply:
Updated.